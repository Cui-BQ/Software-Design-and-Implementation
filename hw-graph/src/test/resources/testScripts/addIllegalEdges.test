# Test add Illegal Edges, e.g. parentNode/childNode/label = null or not exist

# Create a graph
CreateGraph graph1

# Add node n1 and n2
AddNode graph1 n1
AddNode graph1 n2

# Add Illegal edges
AddEdge graph1 null null null
AddEdge graph1 n1 n2 null
AddEdge graph1 n1 null n1-null
AddEdge graph1 null n1 null-n1
AddEdge graph1 n1 new n1-new
AddEdge graph1 new null new-null
