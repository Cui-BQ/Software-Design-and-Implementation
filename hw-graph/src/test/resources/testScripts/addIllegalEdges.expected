# Test add Illegal Edges, e.g. parentNode/childNode/label = null or not exist

# Create a graph
created graph graph1

# Add node n1 and n2
added node n1 to graph1
added node n2 to graph1

# Add Illegal edges
Exception: java.lang.IllegalArgumentException: invalid params
Exception: java.lang.IllegalArgumentException: invalid params
Exception: java.lang.IllegalArgumentException: invalid params
Exception: java.lang.IllegalArgumentException: invalid params
Exception: java.lang.IllegalArgumentException: invalid params
Exception: java.lang.IllegalArgumentException: invalid params